app/actions/addRequest.ts

```diff
@@ -0,0 +1,15 @@
+export const RESET = "RESET";
+export const UPDATE_REQUEST_FIELDS = "UPDATE_REQUEST_FIELDS";
+
+export const updateAddRequestFields = (url: string, method: string, error: string) => {
```

Comment:
This method should accept only one param which is an object. that how we can generalize this method and use instead of n number of functions.